model/project.go

```diff
@@ -1915,7 +1926,9 @@ func (p *Project) extractDisplayTasks(pairs TaskVariantPairs) TaskVariantPairs {
 }
 
 // BuildProjectTVPairsWithAlias returns variants and tasks for a project alias.
-func (p *Project) BuildProjectTVPairsWithAlias(vars []ProjectAlias) ([]TVPair, []TVPair, error) {
+// This filters out tasks that cannot run due to being disabled or having an
+// unmatched requester (e.g. a patch-only task for a mainline commit).
+func (p *Project) BuildProjectTVPairsWithAlias(vars []ProjectAlias, requester string) ([]TVPair, []TVPair, error) {
```

Comment:
While we're at it here can we rename `vars` to something like `aliases` or `projectAliases`?

Reply:
Sure, I renamed it to `aliases` + a couple other places in this file.